test-app/frontend/src/forms/form/Form.tsx

```diff
@@ -0,0 +1,25 @@
+import Form from '@rjsf/material-ui';
+import { JSONSchema7 } from 'json-schema';
+import { useState } from 'react';
+import './Form.scss';
+
+interface IProps {
+  schema: JSONSchema7,
+  onSubmit: (formData: object) => void,
```

Comment:
Not loving the `object` as the type. Can we use generics here instead? On this and perhaps the component. Then just cast the formData for now with `as` or `as unknown as`?

Reply:
Will keep in mind for next time. Since I am simplifying to just pass the event to onSumbit, I will go ahead and type the form event type without any generics.